src/classes/GRER_SyncGradeLevel_TEST.cls

```diff
@@ -97,9 +102,9 @@ private class GRER_SyncGradeLevel_TEST {
         system.assertEquals('Transitional Kindergarten', resultContacts[0].Current_Grade_Level__c, 'The current grade level of the contact is not updated properly');
 
         List<Grade_Enrollment__c> resultGradeEnrollments= [SELECT Status__c FROM Grade_Enrollment__c];
-        system.assertEquals(2, resultGradeEnrollments.size(), 'There should be two grade enrollments created');
+        system.assertEquals(3, resultGradeEnrollments.size(), 'There should be 3 grade enrollments created');
```

Comment:
NIT: just a nit pick here but we should try to be consistent with our syntax, "System.assertEquals" appears to be what we are shooting for elsewhere in the class. (This applies to wherever we have this coded)

Reply:
Fixed it.